api/src/test/java/org/pmiops/workbench/api/AppsControllerTest.java

```diff
@@ -129,6 +129,12 @@ public void testDeleteAppSuccess() {
     verify(mockLeonardoApiClient).deleteApp(APP_NAME, testWorkspace, deleteDisk);
   }
 
+  @Test
+  public void testCanPerformAppActions() {
```

Comment:
I noticed the negative cases for this method are well covered, but not the positive